packages/splits-sdk/src/client/data.ts

```diff
@@ -322,6 +322,8 @@ export class DataClient {
     distributed: FormattedTokenBalances
     activeBalances?: FormattedTokenBalances
   }> {
+    const functionPublicClient = this._publicClient
```

Comment:
i think fine to leave this change. actually probably safer to store it in a local variable, don't risk it getting changed halfway through the function execution.